contents/docs/session-replay/manual.mdx

```diff
@@ -17,95 +14,36 @@ Session replay enables you to record users navigating through your website and p
-
-## Console logs recording
-
-PostHog can also capture console logs from your application. This is useful for debugging and providing extra context on what is happening in your users' browser environment. 
-
-As console logs can contain sensitive information, we do not capture these logs automatically. You can enable this feature globally from your [project settings](https://app.posthog.com/project/settings) or client-side by setting `enable_recording_console_log: true` in our [JavaScript web library config](/docs/integrate/client/js/#config).
+import Snippet from "../../../integrate/\_snippets/install-js-snippet.mdx"
+import Badge from "../../../getting-started/\_snippets/badge.mdx"
+import InstallJSViaNPM from "../../../integrate/\_snippets/install-js-via-npm.mdx"
 
-Console logs will be recorded if _either_ the project setting **or** the client-side config is set to `true`. Console logs will not be recording if session replay is not enabled.
+### Option one: Add JavaScript snippet to your HTML <Badge>Recommended</Badge>
 
-```js-web
-posthog.init('<ph_project_api_key>', {
-    api_host: '<ph_instance_address>',
-    // This is only needed if you aren't configuring session replay using the project settings toggle.
-    enable_recording_console_log: true,
-    // ... other options
-})
-```
+<Snippet />
 
-## Network recording
+### Option two: Install via NPM
 
-PostHog can capture network requests that occur during the browser session so you can see the effect of slow network requests or errors on the user experience.
+<InstallJSViaNPM />
 
-To remove sensitive information from the URL, these network requests can be modified before being captured like so:
+## Step 2: Enable recordings in your project settings
 
-```js-web
-posthog.init('<ph_project_api_key>', {
-    session_recording: {
-        maskNetworkRequestFn: (request) => {
-            // For example: ignoring a request entirely
-            if (request.url.includes('example.com')) {
-                return null
-            }
+Enable session recordings in your PostHog [Project Settings](https://app.posthog.com/project/settings).
 
-            // ... or remove the query string from the URL
-            request.url = request.url.split('?')[0]
-            return request
-        }
-    }
-})
-```
+![Enable session recordings in your PostHog](../../images/docs/session-replay/enable-session-replay-in-project-settings.png)
 
-## How to control which sessions you record
-
-If you want more granular controls on selecting which sessions to record, first set `disable_session_recording: true` in your [config](/docs/integrate/client/js/#config) and then manually start recording by calling `posthog.startSessionRecording()`. Similarly, you can stop the recording at any point by calling `posthog.stopSessionRecording()`.
+Once enabled, the library will start recording sessions by default.
```

Comment:
Should we mention something about sessions recordings being blocked and reverse proxy?

Reply:
We could. There's a "troubleshooting and FAQ section", so I think that's sufficient for now